client/app/pods/components/question-select/component.coffee

```diff
@@ -0,0 +1,8 @@
+`import QuestionComponent from 'tahi/pods/components/question/component'`
+
+QuestionSelectComponent = QuestionComponent.extend
+  selectedData: Em.computed 'model.answer', ->
+    @get('source').findBy 'id', parseInt(@get('model.answer'))
+
+
+`export default QuestionSelectComponent`
```

Comment:
This `QuestionSelect` looks different from what's in the [billing card branch currently](https://github.com/Tahi-project/tahi/blob/1318eae73599eb43927cc5f2312f8acd908dd90d/client/app/pods/components/question-select/component.coffee). Did this additional code get lost in the rebase of this branch?